sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryKMP.kt

```diff
@@ -19,6 +21,8 @@ object Sentry {
      * @param context: The context (used for retrieving Android Context)
      * @param configuration Options configuration handler.
      */
+    @OptIn(ExperimentalObjCRefinement::class)
+    @HiddenFromObjC
```

Comment:
M: Does this require a min. version of KMP that would not work in older versions?
If yes, maybe we can consider just adding a comment that this is not for Apple folks.

Reply:
The min. version for using our KMP sdk is `1.8.0` so nothing would change for current users, I should probably add a badge or something similar to our readme